.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
+
+  coverageReport:
+    strategy:
+      matrix:
+        api-level: [21, 22, 23, 24, 25, 26, 27, 28, 29]
+      fail-fast: false
+    runs-on: macOS-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+
+      - name: create instrumentation coverage
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: ${{ matrix.api-level }}
+          target: google_apis
+          arch: x86_64
```

Comment:
FYI Google [recommends only using x86_64 images for API 26+](https://www.reddit.com/r/androiddev/comments/e5nulf/emulator_29211_and_amd_hypervisor_12_to_stable/f9l6aos?utm_source=share&utm_medium=web2x) 😃

But in my experience I haven't seen any noticeable improvements with x86_64 even on higher API levels.

Reply:
thanks for the pointer @ychescale9